Only in trunk/contrib: no-chdir.patch
diff -ur trunk/onis no-chdir/onis
--- trunk/onis	2005-04-16 00:11:23.000000000 +0200
+++ no-chdir/onis	2005-04-17 14:52:25.000000000 +0200
@@ -15,10 +15,6 @@
 
 BEGIN
 {
-	if ($0 =~ m#^(.*)[/\\]#) { chdir ($1); }
-
-	unshift (@INC, 'lib');
-
 	# 0x0010   Language (make not-translated lines red/yellow)
 	# 0x0020   Parser (dropped lines)
 	# 0x0040   Parser (time information)
